.bit/responses/1.4-Week 1 Step 4.md

```diff
@@ -11,7 +11,7 @@ name: Week 1 Step 4
 
 🕐 Estimated completion: 35-45mins
 
-### Writing Your First Function
+### Building our first function ⚡[HackerVoice]
```

Comment:
This step needs to be rewritten, and the stuff in 1.5 should go here instead. Instead of an HTTP TRIGGER that outputs time, we're asking students to create a trigger that checks for a specific input from the parameter.
1.4: set up azure, get content from request parameter
1.5: use JS to check if password is correct "letmein" - if not, return "Access denied"